src/libraries/System.Security.Cryptography.Cng/tests/SymmetricCngTestHelpers.cs

```diff
@@ -288,10 +288,14 @@ internal static bool SupportsPersistedSymmetricKeys
         internal static byte[] GenerateRandom(int count)
         {
             byte[] buffer = new byte[count];
+#if NETCOREAPP || NETSTANDARD2_1
+            RandomNumberGenerator.Fill(buffer);
+#else
```

Comment:
This is a test. We should prefer simplicity over hyper optimization. This isn't needed.